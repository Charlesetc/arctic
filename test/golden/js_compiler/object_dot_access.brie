define main [
  define x < y = 3 >
  x.y
]
