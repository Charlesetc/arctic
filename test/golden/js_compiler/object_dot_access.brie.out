function main() {
	var x = {y: 3};
	return x.y;
}